wazo_calld/plugins/calls/schemas.py

```diff
@@ -82,6 +82,7 @@ class CallSchema(CallBaseSchema):
     dialed_extension = fields.String()
     sip_call_id = fields.String()
     line_id = fields.Integer()
+    reason_code = fields.Integer()
```

Comment:
`reason_code` is not a part of `Call` object attribute. It should be added only when the event is created